consoleme/lib/cloud_credential_authorization_mapping/__init__.py

```diff
@@ -38,7 +38,7 @@ def __init__(self) -> None:
         self.reverse_mapping = {}
         self.reverse_mapping_last_update = 0
 
-    async def retrieve_credential_authorization_mapping(self):
+    async def retrieve_credential_authorization_mapping(self, max_age=None):
```

Comment:
We should probably add a docstring to clarify that this is the max age of the source data, as opposed to the age of the cached mapping (`self.authorization_mapping_last_update`).

Reply:
Thanks, added docstring